advanced_alchemy/config/common.py

```diff
@@ -181,6 +182,20 @@ class GenericSQLAlchemyConfig(Generic[EngineT, SessionT, SessionMakerT]):
     This is a listener that will automatically save and delete :class:`FileObject <advanced_alchemy.types.file_object.FileObject>` instances when they are saved or deleted.
 
     Disable if you plan to bring your own save/delete mechanism for these columns"""
+    file_object_raise_on_error: bool = False
```

Comment:
IMO this could also default to `True`. While that would be breaking in some sense, I'd say the previous behaviour is a bug, so fixing that bug with how it should actually behave would be fine